app/components/EmptyState/index.js

```diff
@@ -11,37 +11,34 @@ const Container = styled.div`
   align-self: center;
   max-width: 320px;
   padding-top: ${ms(11)};
-`
-
-const Description = styled.p`
-  color: ${ ({ theme: { colors } }) => colors.gray5 };
-  text-align: center;
-`
+`;
 
 const Image = styled.img`(
   display: inline-block;
   padding-bottom: ${ms(4)};
-)`
+)`;
 
 const Title = styled(H6)`
-  color: ${ ({ theme: { colors } }) => colors.primary };
+  color: ${({ theme: { colors } }) => colors.primary};
   margin-top: 20px;
-`
+`;
 
 type Props = {
   title: string,
-  desription: React.Node
-}
+  description: React.Node,
+  imageSrc: string,
+  className: string
```

Comment:
className: string - crashes the app, resulting in empty screen.

![classname-string](https://user-images.githubusercontent.com/37525692/42900384-c780c686-8a96-11e8-898e-669a324b414c.gif)

Reply:
Fixed.